src/dashboard/src/page_manager.c

```diff
@@ -733,45 +733,66 @@ void draw_traction_page(page_manager_t *pm, NHD_US2066_OLED *oled) {
     } else {
         oled_print(oled, DATA_UNKNOWN);
     }
+}
 
 
-    void draw_lc_pid_page(page_manager_t *pm, NHD_US2066_OLED *oled) {
+void draw_lc_pid_page(page_manager_t *pm, NHD_US2066_OLED *oled) {
 
-      carstats_t *stats = pm->stats;
-      static int var_toggled = 1;
+    carstats_t *stats = pm->stats;
+    static int var_toggled = 0;
 
-      // Process contextual actions
-      if (stats->buttons.B.rising_edge) var_toggled++;
-      var_toggled = LOOPOVER(var_toggled, 1, 4);
+    // Process contextual actions
+    if (stats->buttons.B.rising_edge) var_toggled++;
+    var_toggled = LOOPOVER(var_toggled, 0, 2);
 
-      oled_clearline(oled, 0);
-      oled_set_pos(oled, 0, 0);
-      oled_print(oled, "P: ");
-      oled_print(oled, pm->stats->kp);
+    if (stats->buttons.left.action == BUTTON_ACTION_TAP) {
+      switch (var_toggled) {
+        case 0:
+          stats->kp.kp_times_1000 -= 50;
```

Comment:
Might want to enforce a minimum val, but I think the speed controller should take care of accidental neg. inputs